client/src/panels/panel_declarations/results/IndicatorPanel.js

```diff
@@ -102,56 +101,46 @@ const query_api = (id) => {
 export default class IndicatorPanel extends React.Component {
```

Comment:
The name `IndicatorPanel` has become misleading, this is something different now. Maybe `IndicatorModalButton`?

Reply:
A panel's a specific thing, I mean.